src/data/store.js

```diff
@@ -2,14 +2,35 @@ import { applyMiddleware, createStore } from 'redux';
 import thunkMiddleware from 'redux-thunk';
 import { composeWithDevTools } from 'redux-devtools-extension/logOnlyInProduction';
 import { createLogger } from 'redux-logger';
+import { createMiddleware } from 'redux-beacon';
+import Segment, { identifyUser, trackEvent, trackPageView } from '@redux-beacon/segment';
+import { LOCATION_CHANGE } from 'react-router-redux';
+import { FETCH_LOGIN_SUCCESS } from './constants/authentication';
+import { FETCH_CSV_REQUEST } from './constants/courseEnrollments';
 
 import reducers from './reducers';
 
 const loggerMiddleware = createLogger();
 
+const eventsMap = {
```

Comment:
I wonder if we created actions that could be dispatched for sorting/pagination so that we can track those here as well?

Reply:
I think that is what we would have to do in order to track them - I decided not to take on that lift in this PR.

Reply:
Gotcha. Should we create another ticket for the tracking of sorting/pagination then? Or did we decide it's not important to track.

Reply:
I'll create a ticket.